storage/backup/backup-cli/src/backup_types/transaction/tests.rs

```diff
@@ -116,6 +116,18 @@ fn end_to_end() {
     // care of it before running consensus. The latest transactions are deemed "synced" instead of
     // "committed" most likely.
     let tgt_db = AptosDB::new_readonly_for_test(&tgt_db_dir);
+    let ouptputlist = tgt_db
+        .get_transaction_outputs(0, target_version, target_version)
+        .unwrap();
+    assert!(
+        ouptputlist
+            .transactions_and_outputs
+            .iter()
+            .map(|(_, output)| output.write_set().clone())
+            .next()
+            .is_some(),
+        "should have writeset data"
+    );
```

Comment:
I would assert the write sets are actually the same with what's on the txns_to_commit vec